build.cake

```diff
@@ -163,7 +168,8 @@ Task("InstallDotNetCoreSdk")
         // Install legacy .NET Core SDK (used to 'dotnet restore' project.json test projects)
         InstallDotNetSdk(env, buildPlan,
             version: buildPlan.LegacyDotNetVersion,
-            installFolder: env.Folders.LegacyDotNetSdk);
+            installFolder: env.Folders.LegacyDotNetSdk,
+            noPath: true);
```

Comment:
ℹ️ Our build script should be adding the .NET Core SDK that we built with to the path. However, we don't want the legacy .NET Core SDK we're using on the path.